### PR TITLE
Mm/step navigation

### PR DESCRIPTION
**Pull request by MissyM2**
- created skeleton app with home, starting-balance, income, expenses, summary

- added a few images

### CODE REVIEW

File: cfgov/unprocessed/apps/mmt-my-money-calendar/css/main.less [hunk 1]
```diff
@@ -0,0 +1 @@
+@import (reference) "../../../css/main.less";
```
Comment (mgreen-simplethread) [inline cfgov/unprocessed/apps/mmt-my-money-calendar/css/main.less]: Why is this necessary if we're just using existing classes that are already pulled into the page?

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes 2.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
```
Comment (mgreen-simplethread) [inline cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes 2.js]: Did you intend to commit this file? It's a duplicate of routes. Is anything importing this?

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes.js [hunk 3]
```diff
@@ -11,8 +13,16 @@ const Routes = () => (
           <Home />
         </Route>
 
-        <Route path="/wizard">
-          <Wizard />
+        <Route path="/wizard/steps/starting-balance">
```
Comment (mgreen-simplethread) [inline cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes.js]: I thought we were going with numbered step URL segments to simplify back and forward navigation

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/home/hero.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { Link, useRouteMatch } from "react-router-dom";
```
Comment (mgreen-simplethread) [inline cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/home/hero.js]: This is good for now, but we'll eventually want to make this component configurable using props and most likely move this into a shared components folder so that other views can potentially use it.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js [hunk 5]
```diff
@@ -6,18 +6,30 @@ export default function Summary() {
```
Comment (mgreen-simplethread) [inline cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js]: This file needs to be renamed to use a lowercase "i" or it's not going to import correctly

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js [hunk 6]
```diff
@@ -6,18 +6,30 @@ export default function Summary() {
 
   return (
     <section id="summary-view">
-      <h1>Summary Screen</h1>
-
+      <img
+        src="/static/apps/mmt-my-money-calendar/img/4.png"
+        alt=""
+        height="42"
```
Comment (mgreen-simplethread) [inline cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js]: Use CSS to control image dimensions

Comment (mgreen-simplethread) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js]: Actually, nevermind for now. We'll need to figure out a solution for our custom styles, either styled components or loading stylesheets with webpack

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/wizard/index.js [hunk 7]
```diff
@@ -15,11 +18,27 @@ export default function Wizard() {
       <pre className="debug">{JSON.stringify(match, null, 2)}</pre>
 
       <Switch>
-        <Redirect exact from="/wizard" to="/wizard/step/1" />
-        <Route path="/wizard/step/:step">
-          <Step />
+        {/* <Redirect exact from="/wizard" to="/wizard/steps/starting-balance" /> */}
+
+        <Route path="/wizard/steps/starting-balance">
```
Comment (mgreen-simplethread) [inline cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/wizard/index.js]: You have these routes in two different places.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js (hunk 5 above)
Comment (mgreen-simplethread) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js]: Weird, it does import correctly somehow, but I'm not sure why. Still a good idea to rename.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/css/main.less (hunk 1 above)
Comment (MissyM2) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/css/main.less]: Hmm, interesting.  Maybe I don't need it.  I just looked at some of the other satellite apps and they all had a main.less file.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes 2.js (hunk 2 above)
Comment (MissyM2) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes 2.js]: No.  I will clean this stuff up.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes.js (hunk 3 above)
Comment (MissyM2) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes.js]: This is, by no means, a final.  I was having some trouble, short on time and so I just did what I knew how to do.  Let's talk..  I can get some prototyping working but my stuff will definitely have to be refactored.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/home/hero.js (hunk 4 above)
Comment (MissyM2) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/home/hero.js]: Maybe you can do some thinking about the best ways to do things.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js (hunk 5 above)
Comment (MissyM2) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js]: Sorry, I thought I did that.  I will do some cleanup.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js (hunk 6 above)
Comment (MissyM2) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js]: Yes, definitely.  I didn't have time to do the correct stuff.  I know that their styles will take care of a bunch of this.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/wizard/index.js (hunk 7 above)
Comment (MissyM2) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/wizard/index.js]: I changed my methods.  I'll clean up

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js (hunk 5 above)
Comment (MissyM2) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js]: This is weird.  I did change the name of that file.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/css/main.less (hunk 1 above)
Comment (mgreen-simplethread) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/css/main.less]: I don't think this is giving us anything that the main.css file that's already loaded in isn't giving us. Let's get rid of it and figure out a solution for custom styles next week.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes 2.js (hunk 2 above)
Comment (mgreen-simplethread) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes 2.js]: Yeah, just delete this, `git rm`

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes.js (hunk 3 above)
Comment (mgreen-simplethread) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/routes.js]: I'd rather not merge stopgap solutions to the develop branch just so that we can get stuff onto stage for people to see. It would just be making messes we'd need to clean up later - https://en.wikipedia.org/wiki/Technical_debt

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/home/hero.js (hunk 4 above)
Comment (mgreen-simplethread) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/home/hero.js]: Yeah, I'll think about this and we can work on something early next week.

File: cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js (hunk 5 above)
Comment (mgreen-simplethread) [reply cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/summary/Index.js]: Did you use `git mv` to rename it, or did you `git add` it after renaming if you did it a different way?
